Gemfile

```diff
@@ -11,6 +11,7 @@ gem "redis", "~> 4.0"
 gem "sass-rails", ">= 6"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
+gem "mini_magick"
```

Comment:
Would the `image_processing` gem provide the same functionality? It seems to be the [recommended way](https://guides.rubyonrails.org/active_storage_overview.html#transforming-images) to install `mini_magick` for processing attachments.

Reply:
Switch in cdc539f